### ISSUE TITLE
[RFE] render rest endpoint

### ISSUE THREAD

**Issue by DumpySquare** [F5Networks/f5-appsvcs-templates#39]
### Is your feature request related to a problem? Please describe.
User would like to confirm template output without having to send to as3.

### Describe the solution you'd like

a POST to the /mgmt/shared/fast/templates/{setName}/{templateName} with the template parameters should produce the rendered template output, nothing else.

```json
{
  "url": "/mgmt/shared/fast/templates/{setName}/{templateName}",
  "method": "POST",
  "data": {
    "parameters": {
      "tenant_name": "fastTen1",
      "application_name": "app1_http",
      "virtual_port": 80,
      "virtual_address": "10.90.0.10",
      "server_port": 8080,
      "server_addresses": [
        "10.90.0.11",
        "10.90.0.12"
      ]
    }
  }
}
```

### Describe alternatives you've considered
no current alternative

### Additional context
none

**Comment by shyawnkarim** [comment]
This enhancement is being tracked internally with ID MYSTIQUE-328.

**Comment by shyawnkarim** [comment]
Closing.

This enhancement was included with [Release 1.8](https://github.com/F5Networks/f5-appsvcs-templates/releases/tag/v1.8.0).
